platform/core/.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
 /src/flatbuffers_generated
-/flatbuffers
+/flatbuffers_shared/flatc
```

Comment:
I generally dislike hard-coded paths in the .gitignore file. Does it make sense to just put the file name?